TramsDataApi/Migrations/TramsDb/20221208133423_sdd-newcols-datamigrationmk2.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace TramsDataApi.Migrations.TramsDb
+{
+    public partial class sddnewcolsdatamigrationmk2 : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
There are no migrations for the new columns - DynamicsSchoolLeaseId, DynamicsSchoolLoanId & DynamicsKeyPersonId